pkg/openstack/common.go

```diff
@@ -291,11 +294,26 @@ func EnsureEndpointConfig(
 						}
 						return endpoints, ctrl.Result{}, err
 					}
+					// Delete the issued certificate if it exists
+					cert := certmanager.NewCertificate(
+						&certmgrv1.Certificate{
+							ObjectMeta: metav1.ObjectMeta{
+								Name:      ed.Route.TLS.CertName,
+								Namespace: ed.Namespace,
+							},
+						},
+						5*time.Second,
+					)
+					err = cert.Delete(ctx, helper)
+					if err != nil {
+						helper.GetLogger().Error(err, fmt.Sprintf("Failed to delete unused route certificate %s", ed.Route.TLS.CertName))
```

Comment:
its a duplicate error message as we already get the one from https://github.com/openstack-k8s-operators/lib-common/blob/main/modules/certmanager/certificate.go#L160, do we need this?